frontend/resourceadm/components/PartyListDetails/PartyListDetail.test.tsx

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import { render as rtlRender, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { act } from 'react-dom/test-utils';
+import { textMock } from '../../../testing/mocks/i18nMock';
+import { queriesMock } from 'app-shared/mocks/queriesMock';
+import { PartyListDetail, PartyListDetailProps } from './PartyListDetail';
+import { ServicesContextProps, ServicesContextProvider } from 'app-shared/contexts/ServicesContext';
+import { createQueryClientMock } from 'app-shared/mocks/queryClientMock';
+
+const mockedNavigate = jest.fn();
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useNavigate: () => mockedNavigate,
+}));
+
+const testOrg = 'ttd';
+const testEnv = 'tt02';
+const testListeIdentifier = 'listid';
+
+const defaultProps = {
+  org: testOrg,
+  env: testEnv,
+  list: {
+    env: testEnv,
+    identifier: testListeIdentifier,
+    name: 'Test-liste',
+    description: 'Dette er en beskrivelse',
+    members: [
+      {
+        orgNr: '123456789',
+        orgName: '',
+        isUnderenhet: false,
+      },
+    ],
+  },
+  backUrl: '/listadmin',
+};
+
+const user = userEvent.setup();
+
+describe('PartyListDetail', () => {
+  it('should show special name if name is not found', () => {
+    render();
+    expect(screen.getByText(textMock('resourceadm.listadmin_empty_name'))).toBeInTheDocument();
+  });
+
+  it('should show message when list is empty', () => {
+    render({ list: { ...defaultProps.list, members: [] } });
+    expect(screen.getByText(textMock('resourceadm.listadmin_empty_list'))).toBeInTheDocument();
+  });
+
+  it('should call service to remove member', async () => {
+    const removePartyListMemberMock = jest.fn();
+    render({}, { removePartyListMember: removePartyListMemberMock });
+
+    const removeButton = screen.getByText(textMock('resourceadm.listadmin_remove_from_list'));
+    await act(() => user.click(removeButton));
+
+    expect(removePartyListMemberMock).toHaveBeenCalled();
```

Comment:
Can we check that it has been called ```x``` amount of times or with ```x``` data instead to make sure the functions are not being called too many times or with the wrong data? 😄 
This goes for the other test files as well.